### ISSUE TITLE
Add support for Lighthouse

### ISSUE THREAD

**Issue by medvedev1088** [blockchain-etl/ethereum2-etl#4]

**Comment by lsankar4033** [comment]
Given that infura's only running teku for now, this would require running a lighthouse node and adapting service code to use lighthouse APIs, right?

**Comment by medvedev1088** [comment]
That's correct. Most likely though Lighthouse API is very similar to Teku so not much change will be required.

**Comment by lsankar4033** [comment]
gotcha. given that you're not pulling performance-oriented data from what i can tell, probably not particularly high priority to add more clients

**Comment by medvedev1088** [comment]
Yep we discussed with Steven that we'll only do immutable data for now due to BigQuery limitations
